hibernate-reactive-core/src/main/java/org/hibernate/reactive/adaptor/impl/ResultSetAdaptor.java

```diff
@@ -633,7 +641,10 @@ public Ref getRef(String columnLabel) {
 
 	@Override
 	public Blob getBlob(String columnLabel) {
-		throw new UnsupportedOperationException();
+		Buffer buffer = (Buffer) row.getValue( columnLabel );
+		return ( wasNull = buffer == null )
+				? null
+				: BlobTypeDescriptor.INSTANCE.wrap( buffer.getBytes(), options );
```

Comment:
@DavideD so this looks like a layer-breaker to me.

I think this should just be a call directly to `BlobProxy.generateProxy()`, and then we can eliminate the passing of `WrapperOptions`.

Reply:
I've updated the commit: https://github.com/hibernate/hibernate-reactive/pull/290/commits/08fe85f26d05cc639e865ed09fab9953fded0acc

Reply:
Thanks.